### DISCUSSION TITLE
Running Mobile Automation Scripts using  Robot frame work,

### DISCUSSION THREAD

**Discussion by sekharpadala333** [Genymobile/feedback#168] [Genymotion Device Image] [answered]
Does the  Genymotion Device image support **RobotFrame**

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @sekharpadala333 , this board is not suitable for this type of query.
Please submit a ticket to our support team at https://support.genymotion.com/hc/en-us/requests/new?ticket_form_id=83539

Thank you for your understanding
